test/e2e/workflow/deployment_test.go

```diff
@@ -160,6 +169,39 @@ var _ = Describe("NetworkAddonsConfig", func() {
 			CheckConfigCondition(ConditionAvailable, ConditionTrue, 10*time.Minute, 30*time.Second)
 		})
 	})
+
+	//TODO: remove Eventually and just use GetConfig after issue 141 is resolved
+	// (https://github.com/kubevirt/cluster-network-addons-operator/issues/141)
+	FContext("testing range generation after kubeMacPool reinstalation ", func() {
```

Comment:
Drop the focus, it skips all the rest of our tests.

Reply:
oops